src/system/util.ts

```diff
@@ -349,3 +352,29 @@ export function activateChatTab() {
   // @ts-ignore 0.8 tabs
   ui?.sidebar?.tabs.chat && ui.sidebar?.activateTab(ui.sidebar.tabs.chat.tabName);
 }
+
+
+export function getDefaultRqid(Item: RqgItem): string {
+  if (Item.type === ItemTypeEnum.Skill) {
```

Comment:
I think if you check
```suggestion
  if (Item.data.type === ItemTypeEnum.Skill) {
```
then you don't need to do the assertItemType to get to make item narrowed to a skill item. Same in the other if:s

Reply:
done